src/tests/ftest/util/job_manager_utils.py

```diff
@@ -550,6 +563,16 @@ def wait(self):
         """Wait for the sub process to complete."""
         raise NotImplementedError()
 
+    def kill(self):
+        """Forcibly terminate any job processes running on hosts."""
+        try:
+            self.stop()
+        except CommandFailure as error:
+            self.log.info(
+                "Error stopping/disabling {}: {}".format(
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments

Reply:
Fixed